docs/gut-to-soil-16S-tutorial.md

```diff
@@ -477,16 +477,22 @@ Here are the steps that it takes:
 ::::{margin}
 :::{note}
 When using [q2-kmerizer](https://library.qiime2.org/plugins/bokulich-lab/q2-kmerizer), normalization should be done at the ASV level before kmerization, not at the kmer level.
-This is automated by the [`kmer-diversity`](xref:q2-boots-target#q2-action-boots-kmer-diversity) that we're using here.
+This is automated by the [`kmer-diversity`](xref:q2-boots-target#q2-action-boots-kmer-diversity) action that we're using here.
 This is because you are trying to normalize by sequencing depth, not kmer complexity.
 In the end, the difference should not be huge but this distinction could be important for some metrics.
 :::
 ::::
 
+::::{margin}
+:::{note}
+[q2-kmerizer](https://library.qiime2.org/plugins/bokulich-lab/q2-kmerizer) decomposes each sequence into its constituent kmers. This should be carefully considered when interpreting alpha-diversity in particular, as the number of observed features (for example) would correspond to the number of unique kmers observed in a sample (representing the genetic diversity), not the number of unique sequences or taxa. For more information, read the [q2-kmerizer](https://doi.org/10.1128/msystems.01550-24) paper.
```

Comment:
@nbokulich, it's a little odd to get used to, but when writing these we now go with the convention of one sentence per line. That makes reviewing diffs much easier. (Single line breaks are ignored by the renderers - double line breaks make new paragraphs.)

```suggestion
[q2-kmerizer](https://library.qiime2.org/plugins/bokulich-lab/q2-kmerizer) decomposes each sequence into its constituent kmers. 
This should be carefully considered when interpreting alpha-diversity in particular, as the number of observed features (for example) would correspond to the number of unique kmers observed in a sample (representing the genetic diversity), not the number of unique sequences or taxa. 
For more information, read the [q2-kmerizer](https://doi.org/10.1128/msystems.01550-24) paper.
```